tests/web_service/test_edge_cases.py

```diff
@@ -0,0 +1,60 @@
+"""
+Test edge cases
+"""
+
+import logging
+
+from pydantic import ValidationError
+from pytest import skip
+
+from poprox_concepts.api.recommendations import RecommendationRequest
+from poprox_recommender.config import allow_data_test_failures
+from poprox_recommender.paths import project_root
+
+logger = logging.getLogger(__name__)
+
```

Comment:
For these tests, I think we'll get more mileage out of them if we integrate calls to the recommenders (like you mentioned) so that we can verify that these edge cases don't break the recommendation pipelines. Let's talk about how to do that!